### ISSUE TITLE
create new vector shapefile not working

### ISSUE THREAD

**Issue by eichners** [qgis/QGIS#31186]
Layer --> new shapefile --> polygon or line

QGIS 3.4 installed on imac, OS 10.14.5

When creating new shapefile, adding a line vector does not work. 
There is no option to save changes, and no way to finish the line and edit it. Lines appear when I use the editing tool but they can not be selected and do not allow me to delete. Layer does not allow for deletion or saving. When I open the table, no fields are there, no id or other fields I've tried to add.

I've tried reinstalling QGIS and there is no change. 

I do not know if I'm installing QGIS 3.4 correctly -- I'm using the packaged install but I can not find a way to a readme file to be sure I'm doing the install correctly nor if I should install python3.6 separately.

I'm not a programmer so please let me know if I should report a second issue re. the install readme file. I have spent time looking for similar issues on line but don't see anything. Apologies if this is something I'm doing wrong.

**Comment by roya0045** [comment]
Are you in editing mode?

**Comment by eichners** [comment]
Yes 
I’m able to do the same processes on a different Mac.

**Comment by eichners** [comment]
Hi Alex,

A couple more things; 
when I create a new shapefile, I’m able to use the line editing tool and create vertices and lines, but when I right click to save the shape, I’m unable to. The tool just keeps making more vertices. I’m then unable to select or delete the lines that are there and nothing gets saved to the shapefile. 

I am curious if there is anything going on with the recent installation package for QGIS 3.4 for Macs. And in this case, I may be missing some steps if there is more to it than dragging the QGIS file to the applications folder to do the install. There is no readme file in the dmg folder for reference or more detailed instructions. 

I can provide screen shots on Monday. 

Thanks for following up with your email earlier. 
Best,
Sara



Sara Eichner
---------------------------
eichnersara@gmail.com
www.saraeichnergis.com <http://www.saraeichnergis.com/>
rte.nyc <http://www.rte.nyc/>
646-409-9628

**Comment by gioman** [comment]
> A couple more things; when I create a new shapefile, I’m able to use the line editing tool and create vertices and lines, but when I right click to save the shape, I’m unable to. The tool just keeps making more vertices. I’m then unable to select or delete the lines that are there and nothing gets saved to the shapefile. I am curious if there is anything going on with the recent installation package for QGIS 3.4 for Macs. And in this case, I may be missing some steps if there is more to it than dragging the QGIS file to the applications folder to do the install. There is no readme file in the dmg folder for reference or more detailed instructions. I can provide screen shots on Monday.

this is likely to be a local issue, please try using a new/clean QGIS profile, as a start to try troubleshoot your problem.

**Comment by roya0045** [comment]
A gif would be helpful, a tool like screentogif but for mac would help because its hard to understand the steps that you are taking and where things are breaking.

**Comment by eichners** [comment]
I’m unable to make a small enough gif to get through your email size restrictions. 
You can see it on dropbox instead, a slightly larger file than I was trying to send.
link here. <https://www.dropbox.com/s/kjn4xdsyhe266ik/QGIS_vector_edit_problems_largergif.gif?dl=0>

Thank you for taking a look.

-Sara


> 
> 
> Sara Eichner
> ---------------------------
> eichnersara@gmail.com <mailto:eichnersara@gmail.com>
> www.saraeichner.com <https://www.saraeichner.com/>
> www.saraeichnergis.com <https://www.saraeichnergis.com/>
> 
>

**Comment by gioman** [comment]
> Thank you for taking a look.

does it happen also on a clean project? have you tried with a new/clean QGIS profile?

**Comment by eichners** [comment]
This is a new installation of QGIS 3.4 that I installed on Friday. 
The user profile I’ve been using is default.
I created an alternate/ clean user profile called sara.
I created a new file and followed the same steps and exactly the same thing happens. no improvement. 

I have not restarted the computer which I can try now.

When I look at the user profile folders there is very little in them because QGIS was just reinstalled but see the screen shots here:

**Comment by gioman** [comment]
have you tried on a completely empty project?

**Comment by eichners** [comment]
Yes, that’s what I just did.
same problems.

Sara Eichner
---------------------------
eichnersara@gmail.com
www.saraeichnergis.com <http://www.saraeichnergis.com/>
rte.nyc <http://www.rte.nyc/>
646-409-9628

**Comment by eichners** [comment]
I tried the same process with an empty project on my mac laptop. That copy of QGIS 3.4 was installed a few months ago. 

It works fine — just as it should I get the table window with an option to save the vector feature and add a name. I’m able to save the layer and turn it off and on and the attribute table shows the new features.

This problem started on my desktop computer when I reinstalled QGIS 3.4 last week.
Could this have something to do with the installation package or procedures I was following based on the new bundled installer for Macs? Do I need to install python 3.6? 

thanks,
Sara
